test/testdata/parser/whitequark/test_ruby_bug_11873_a_16.rb

```diff
@@ -0,0 +1,3 @@
+# typed: true
+
+a b{c d}, 1.0i do end
```

Comment:
why do these files sometimes have end of line and sometimes don't?

Reply:
by EOL you mean an empty trailing newline, right? If so all files have it:
```
        File.write(input_filepath, "# typed: true\n\n" + capture[:input] + "\n")

        if capture[:output]
          File.write(output_filepath, capture[:output] + "\n")
        end
```